docs/content/concepts/scenarios/kind-scenario/yamls/2-synctarget-kind.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: workload.kcp.io/v1alpha1
+kind: SyncTarget
+metadata:
+  annotations:
+    kcp.io/cluster: br3qimk2t1o3jb70
+  creationTimestamp: "2023-03-21T17:05:37Z"
+  generation: 1
+  labels:
+    internal.workload.kcp.io/key: alIviEyJiysDMorx9fWmI2ygTHSTrvxXGyVwSJ
+  name: kind
+  resourceVersion: "9104"
+  uid: 8df305e7-f4f8-439f-9115-51054d519361
+spec:
+  supportedAPIExports:
+  - export: kubernetes
+    path: root:compute
+  unschedulable: false
+status:
+  conditions:
+  - lastTransitionTime: "2023-03-21T17:05:37Z"
+    message: No heartbeat yet seen
+    reason: ErrorHeartbeat
+    severity: Warning
+    status: "False"
+    type: Ready
+  - lastTransitionTime: "2023-03-21T17:05:37Z"
+    message: No heartbeat yet seen
+    reason: ErrorHeartbeat
+    severity: Warning
+    status: "False"
+    type: HeartbeatHealthy
+  syncedResources:
+  - identityHash: <id-hash>
```

Comment:
I think this is not something we need to redact.  It is not redacted in all cases in the current version of this PR.

I am currently trying to understand how this part of APIExport and APIBinding work, so redacting this is problematic for me.

Reply:
https://docs.kcp.io/kcp/main/concepts/quickstart-tenancy-and-apis/#apis-faq says that the identity kept in that secret is a big random number.  So if your data is from an ephemeral environment that is gone now then revealing that random number is surely harmless.